lib/config.js

```diff
@@ -8,5 +8,6 @@ export default {
     channel: '#' + process.env.SLACK_CHANNEL, // Slack channel without # prefix
     username: process.env.SLACK_USERNAME || 'MyAwesomeBot',
     githubUser: process.env.GITHUB_USER,
-    githubToken: process.env.GITHUB_ACCESS_TOKEN
+    githubToken: process.env.GITHUB_ACCESS_TOKEN,
+    extMode: process.env.EXT_MODE
```

Comment:
That is a good idea. This way the old method would be the default option and if someone places the `google-spreadsheet-config.json` file in the server the new method is activated. Seems fine.